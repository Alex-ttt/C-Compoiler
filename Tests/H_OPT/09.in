void fo(int a, float b, char c){}

void main(){
	const int f = 5.5;
	fo(f, 10 - 9, 'a'); 	
}