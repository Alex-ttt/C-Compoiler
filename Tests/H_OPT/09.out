 Symbols Table:
   fo  void function fo(
          a int
          b float
          c char
  )
    {
      << block is empty >>
    }
   main  void function main(
  )
    {
      Statements:
      fo
      (
        f
      ,
        float        (
            10
          -
            9
        )
      ,
        'a'
      )
      Symbols Table:
              f const int
    }
